http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -3411,6 +3411,12 @@ components:
         - auditInfo
       type: object
       properties:
+        appName:
+          type: string
+          description: the name of the app
+        cloudContext:
```

Comment:
[this](https://github.com/DataBiosphere/leonardo/blob/develop/core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/JsonCodec.scala#L93) is encoder for `cloudContext`, it will be encoded into sth like
```
"cloudProvider": "",
"cloudResource":"",
```

Reply:
I was following the cloudContext in ListAppResponse below: https://github.com/DataBiosphere/leonardo/pull/2911/files#diff-6fad074e668b9395b43409265ad159ab303e3e3ff3c3d63bff4595706a3da88fR3457-R3459
Is that piece also wrong?

Reply:
that link is just diff...which line you're referring to?

Reply:
That link is ListAPP's cloudContext.  Why those are in string format? is that correct? 
https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/swagger/api-docs.yaml#L3448
https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/swagger/api-docs.yaml#L2548

Reply:
I can't speak to the runtime one..but listApp is definitely wrong..

Reply:
ah, okay. Create a `CloudContext`. PTAL.  One thing I am not sure is the case sensitive enum mapping.
Not sure if [CloudProvider.Azure ](https://github.com/DataBiosphere/leonardo/blob/develop/core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/JsonCodec.scala#L352) can be mapped to `AZURE`